### DISCUSSION TITLE
Starting off with WinGET, have some questions!

### DISCUSSION THREAD

**Discussion by Zeppperoni** [microsoft/winget-cli#4847] [General] [answered]
Hi folks, had some quick questions related to WinGET! 

1. Is it possible to always be behind by 1 version of the latest stable release of any package?
2. Can you install from an internal trusted source instead of the external ones provided by Microsoft? Like a private repo

Many thanks!

**Comment by denelon** [comment, marked as answer]
1. That isn't a feature in WinGet yet, but you can query for available versions and use an available version from that output.
2. Take a look at https://github.com/microsoft/winget-cli/restsource. Note: We're about to release a new version which will fix the cmdlet used to deploy the source. It is currently not functional due to some upstream breaking changes, but we're adding Entra ID support and the next release should have that fixed.

There are a couple of commercial and open-source options available as well.
